i18n/en/docusaurus-plugin-content-docs/current/intro.mdx

```diff
@@ -1,94 +1,15 @@
 ---
 sidebar_position: 1
+slug: /
 pagination_next: get-started/index
 ---
 
-# 🔎 Intro
```

Comment:
> Roll back the edits to the intro. Yeah, I'm definitely saying this because I wrote the previous version, but I truly believe we had it quite nicely already, I don't really see a reason to change it

Its more complicated question, yeah 😄 

![image](https://user-images.githubusercontent.com/42924400/188974846-d0343aa5-b3f5-4922-b229-b10b05f5c5c4.png)

According our last discussion about docs problems (screenshot above), we decided to setup "Table of Content" page as IndexPage of whole documentation. Also we solved that for this we should disassemble Intro: smth to landing, smth to GetStarted

**And my main point** - we have TOO MANY sections for docs, and Intro (that should help to "get start") have similar responsibilities as `/get-started` section

As a result, we have:
- less redundant sections
- refactored and single "Overview" article on `/get-started` (next, we'll simplify access it like "one-click")
- simple "IndexPage" of docs (a.k.a ToC) with possibilities to refine
   (now it not depend from old "Intro" content)

Reply:
@unordinarity @illright @Imperyall @Rin-Akaia-eth What do u think?

Reply:
It's hard for me to say here what do I think) I agree that we have too many sections and that some of them have similar responsibilities.

I just don't really understand what the comment in this thread refers to in this PR. What change is it describing?

Reply:
> I just don't really understand what the comment in this thread refers to in this PR. What change is it describing?

Ok, lets discuss at next call 🤷

Reply:
fixed